libs/validation/src/lib/decorators/number/IsCompositeIdentifier.ts

```diff
@@ -0,0 +1,46 @@
+import { buildMessage, ValidateBy, ValidationOptions } from 'class-validator';
+
+// The following should probably be part of our types lib
```

Comment:
Tag the story here.

Reply:
[Here](https://www.pivotaltracker.com/story/show/182694263) is the story, which is now tagged in the comment.